README.md

```diff
@@ -192,6 +194,16 @@ optional arguments:
   -h, --help             show this help message and exit
 ```
 
+### Skip An Image
+
+In case you want the tool to skip updating a particular image tag then add a comment `no-dfiu` after the `FROM` declaration in the Dockerfile. The tool will process the comment following `FROM` declaration and if `no-dfiu` is mentioned, pull request for that image tag will be ignored. You can use an alternate comment string by passing an additional command line parameter `-x IGNORE_PR_COMMENT`. In that case string mentioned with the parameter, will be used for skipping PR creation.
```

Comment:
```suggestion
In case you want the tool to skip updating a particular image tag then add a comment `no-dfiu` after the `FROM` declaration in the Dockerfile. The tool will process the comment following `FROM` declaration and if `no-dfiu` is mentioned, pull request for that image tag will be ignored. You can use an alternate comment string by passing an additional command line parameter `-x IGNORE_IMAGE_STRING`. In that case string mentioned with the parameter, will be used for skipping PR creation.
```

Reply:
Done